chain/jsonrpc-primitives/src/types/config.rs

```diff
@@ -10,7 +16,7 @@ pub struct RpcProtocolConfigRequest {
 #[derive(Serialize, Deserialize, Debug)]
 pub struct RpcProtocolConfigResponse {
     #[serde(flatten)]
```

Comment:
If we are `#[serde(flatten)]` this anyway,do we need a separate `ProtocolConfigView` struct?

Reply:
Yeah, I thought about that, but thought I'd retain an interface downstream users were already familiar with. But you're right. Since we're redesigning the crate anyway, might as well go all the way.